Update `with_payload` examples, also describe using an array directly (#343)
qdrant/landing_page: qdrant-landing/content/documentation/concepts/search.md

)
```

> The parameter `with_payload` may also be used to scope to or filter out a
> specific payload subset. You can specify an array of items to include. For
> example, to include `city`:

Reviewer: I'd rewrite the whole page in [active voice](https://developers.google.com/style/voice) to tell make a reader think "This is what I can do". Here's how you can set up this paragraph with active voice:

```suggestion
You can use `with_payload` to scope to or filter a specific payload subset. 
You can even specify an array of items to include. For example, to include `city`, 
`village`, and `town`:
```

```http
POST /collections/{collection_name}/points/search

{
    "vector": [0.2, 0.1, 0.9, 0.7],

>     "with_payload": ["city"]

Reviewer: Since you suggested that users can specify an array of items (line 213), I'd include an array `with_payload`, maybe something like:

```suggestion
    "with_payload": ["city", "village", "town"]
```
